astropy/coordinates/tests/test_distance.py

```diff
@@ -49,7 +48,7 @@ def test_distances():
 
     # Coordinate objects can be assigned a distance object, giving them a full
     # 3D position
-    c = Galactic(l=158.558650*u.degree, b=-43.350066*u.degree,
+    _ = Galactic(l=158.558650*u.degree, b=-43.350066*u.degree,
```

Comment:
This is a weird one, not checking the result. Strictly, no need to assign anything, although I think I'd prefer a quick `assert c.distance == 12*u.pc`

Reply:
Done.